packages/macro/package.json

```diff
@@ -49,11 +49,11 @@
   "dependencies": {
     "@babel/runtime": "^7.20.13",
     "@babel/types": "^7.20.7",
-    "@lingui/cli": "4.0.0-next.3",
-    "@lingui/conf": "4.0.0-next.3"
+    "@lingui/conf": "4.0.0-next.3",
+    "@lingui/core": "4.0.0-next.3",
```

Comment:
Is this dependency needed here? I don't see any `core` usage inside the `macro` package

Reply:
typings